protocol/src/main/scala/sbt/internal/bsp/BuildServerConnection.scala

```diff
@@ -63,7 +63,7 @@ object BuildServerConnection {
     // For those who use an old sbt script, the -Dsbt.script is not set
     // As a fallback we try to find the sbt script in $PATH
     val fileName = if (Properties.isWin) "sbt.bat" else "sbt"
-    val envPath = sys.env.getOrElse("PATH", "")
+    val envPath = sys.env.get("PATH").orElse(sys.env.get("Path")).getOrElse("")
```

Comment:
Instead of singling out `PATH` and `Path`, would it make sense at this point to filter to all keys that becomes `PATH` if you turned it to upper case, and pick the first entry?

Reply:
Yep, it makes sense. Changed

Reply:
@martingd 
> We already have special handling for Windows for the name of the sbt launcher

Could you point where is this handling defined?
I think the best option would reuse/copy-paste the existing logic